support/jsdoc/theme/static/styles/jsdoc-default.css

```diff
@@ -127,12 +143,32 @@ section, h1 {
 
 .nav.navbar-right .navbar-form {
     padding: 0;
-    margin: 0;
+    margin: 6px 0px;
+}
+
+.navbar-collapse.collapse {
+  display: block!important;
+}
+
+.navbar-nav>li, .navbar-nav {
+  float: left !important;
+}
+
+.navbar-nav.navbar-right:last-child {
+  margin-right: -15px !important;
+}
+
+.navbar-right {
+  float: right!important;
```

Comment:
Those `!important` are to override the default bootstrap behavior..
[http://stackoverflow.com/a/23536146/1234504](http://stackoverflow.com/a/23536146/1234504)